lib/modules/showImages.js

```diff
@@ -1744,6 +1749,54 @@ addModule('showImages', function(module, moduleID) {
 		}
 	}
 
+	function setMediaControls(mediaTag) {
+
+		if (!module.options.imageControls.value) return;
+
+		// Controls to show
+		var control_buttons = {
+			'rotateLeft': 'Rotate image counter-clockwise',
+			'rotateRight': 'Rotate image clockwise'
+		};
+
+		// generate controls bar
+		var controls = document.createElement('div');
+		controls.className = 'RESMediaControls';
+
+		// Add controls from array
+		var tmp, i;
+		for (i in control_buttons){
+			tmp = document.createElement('span');
+			tmp.className = i;
+			tmp.title = control_buttons[i];
+			tmp.setAttribute('data-action', i);
+			controls.appendChild(tmp);
+		}
+
+		// Rotation controls are only for images (as videos will end up with their controls the wrong way up)
+		var media = $(mediaTag);
+		media.data('media-controls', controls);
+		$(mediaTag).parent().append(controls);
+
+		// Hook up controls
+		$(controls).find('span').click(function(e){
```

Comment:
Hmm, just gave this a try, but it doesn't seem like the delegated events are able to stop the click propagating to the anchor wrapping em.